### PR TITLE
livepatch: Add full path to all cmds (SC-627)

### PR DESCRIPTION
**Pull request by lucasmoura**
## Proposed Commit Message
livepatch: Add full path to all cmds

When interacting with livepatch, if the user does not have /snap/bin in the PATH, the pure canonical-livepatch command
will fail. We are now specifying the full path for all livepatch commands we are running.

LP: #1951954

## Test Steps
Run any integration test that enable/disables livepatch

## Desired commit type

 - [ ] Squash and merge: Using the "Proposed Commit Message"
 - [ ] Create a merge commit and use "Proposed Commit Message"
 - [ ] Rebase and merge, no merge commit, rely only on existing commit messages

## Checklist

 - [ ] I have updated or added any unit tests accordingly
 - [ ] I have updated or added any integration tests accordingly
 - [ ] I have updated or added any documentation accordingly

### CODE REVIEW

**Review by orndorffgrant — COMMENTED** [review]
Thanks for this fix! I'm running some integration tests now. The only other place we could use this is in the collect-logs command where we get the livepatch status.

**Comment by orndorffgrant** [comment]
And my tests passed! Will approve after update to collect-logs :)

**Comment by lucasmoura** [comment]
@orndorffgrant Good catch. I have updated the code

**Review by renanrodrigo — APPROVED** [review]
lgtm

**Review by orndorffgrant — APPROVED** [review]
Just the smallest of nits. Otherwise LGTM

**Comment by orndorffgrant** [comment]
Oof didn't save my comment, one sec

File: uaclient/cli.py [hunk 1]
```diff
@@ -1143,7 +1143,7 @@ def action_collect_logs(args, *, cfg: config.UAConfig):
             "ua status --format json", "{}/ua-status.json".format(output_dir)
         )
         _write_command_output_to_file(
-            "canonical-livepatch status",
+            "/snap/bin/canonical-livepatch status",
```
Comment (orndorffgrant) [inline uaclient/cli.py:1146]: Tiny nit: Should this use the `livepatch.LIVEPATCH_CMD` variable?

Comment (lucasmoura) [reply uaclient/cli.py:1146]: I thought about that, but I did not want to add a direct entitlement import into the cli module.
But maybe this doesn't matter that much.

What do you think ?

Comment (orndorffgrant) [reply uaclient/cli.py:1146]: @lucasmoura Good point, that is a sign that something is a little weird.

I think its a sign that the general livepatch functionality that isn't directly tied to the entitlement should be refactored into its own module maybe? Not urgent though.

Up to you on whether you think its reasonable or not. I think I still lean towards using the constant, but am going to go ahead and approve

**Review by orndorffgrant — APPROVED** [review]

File: uaclient/cli.py (hunk 1 above)
Comment (lucasmoura) [reply uaclient/cli.py:1146]: Yes, currently we have no way to handle service specifc requests anywhere in the codebase.
I agree that we need a refactoring on that part. I will use the constant and add a comment that should touch this in the near future

**Pull request merged**